src/encoded/types/functional_characterization_experiment.py

```diff
@@ -234,6 +297,36 @@ def datapoint(self, request, related_series=None):
                 return True 
         return False
 
+    @calculated_property(schema={
+        "title": "Biosamples",
+        "type": "array",
+        "items": {
+            "type": 'string',
+            "linkTo": "Biosample",
+        },
+        "notSubmittable": True,
+    })
+    def biosamples(self, request, replicates=None):
```

Comment:
Is this function exactly the same for series and experiment?  Is there any way to share the function?